src/compaction.c

```diff
@@ -26,9 +26,9 @@ typedef struct AvgContext {
 } AvgContext;
 
 typedef struct StdContext {
-    double std;
-    double mean;
-    double cnt;
+    double sum;
+    double sum_2;
```

Comment:
sum_2 will be confusing for a future reader. Maybe add a comment to explain more

Reply:
comment added